modules/labs-react/expandable/lib/hooks/useExpandableTarget.ts

```diff
@@ -1,6 +1,9 @@
 import {createElemPropsHook} from '@workday/canvas-kit-react/common';
 import {useExpandableModel} from './useExpandableModel';
 
+/**
```

Comment:
Can we leave this off or was this intentional?

Reply:
It isn't needed. There was no JSDoc in the MDX file.